docs/Early Access - Using Global Profile Configuration.md

```diff
@@ -57,18 +60,24 @@ To get started, install the Zowe CLI `@next` version from the online registry. Y
 
    Zowe CLI and optional plug-ins are installed!
 
-5. If you previously had an instance of Zowe CLI installed, your old configuration files are no longer used in this version. Delete the following files from your local `.zowe/` directory:
-   - `.zowe/settings/imperative.json`
-   - `.zowe/profiles`
+5. Open a command-line window and issue the following command:
 
-   **Important!** Prior to deleting the contents of the `/profiles` directory, take note of any mainframe service details that you need (host, port, etc...). You might want to save the entire `/profiles` directory to another location on your computer so that you can reference or restore the profiles later.
+   ```
+   zowe scs revert --force
```

Comment:
Perhaps we should:
1. inform users that this will remove old secure credentials stored in their operating system's credential vault (the desired behavior)
1. warn users that this also reverts all securely stored credentials to being stored in plain text (potentially undesirable side effect and could be bad if they neglect to delete `~/.zowe/profiles`)